server/src/test/java/org/elasticsearch/cluster/node/DiscoveryNodesTests.java

```diff
@@ -254,6 +262,13 @@ private static DiscoveryNode newNode(int nodeId, Map<String, String> attributes,
             Set<String> matchingNodeIds(DiscoveryNodes nodes) {
                 return Collections.singleton(nodes.getMasterNodeId());
             }
+        }, COORDINATING_ONLY("_coordinating_only") {
```

Comment:
This looks like a hangover of the previous approach - should be `coordinating_only:true` I think.